examples/gallery/embellishments/pygmt_logo.py

```diff
@@ -0,0 +1,256 @@
+
+
+# %%
+
+# -----------------------------------------------------------------------------
+# Start plotting
+# -----------------------------------------------------------------------------
+
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Creating the visual
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+fig = pygmt.Figure()
+pygmt.config(MAP_FRAME_PEN="cyan@100")
+fig.basemap(region=region, projection=f"X{size * 2}c", frame=[0, "+gcyan@100"])
+
+# .............................................................................
+# blue circle / hexagon for Earth
+# .............................................................................
+match shape:
+    case "circle":
+        style = "c7.5c"
+    case "hexagon":
+        style = "h8.6c"
+fig.plot(x=0, y=0, style=style, pen=f"15p,{color_blue}", fill=color_bg)
+
+# .............................................................................
+# yellow lines for compass
+# .............................................................................
+# horizontal yellow line
+fig.plot(x=[-4, 4], y=[0, 0], pen=pen_yellow)
+# diagonal yellow lines
+# upper left
+fig.plot(x=[-xy_yellow_1, -xy_yellow_2], y=[xy_yellow_1, xy_yellow_2], pen=pen_yellow)
+# lower right
+fig.plot(x=[xy_yellow_2, xy_yellow_1], y=[-xy_yellow_2, -xy_yellow_1], pen=pen_yellow)
+# lower left
+fig.plot(x=[-xy_yellow_1, -xy_yellow_2], y=[-xy_yellow_1, -xy_yellow_2], pen=pen_yellow)
+# upper right
+fig.plot(x=[xy_yellow_2, xy_yellow_1], y=[xy_yellow_2, xy_yellow_1], pen=pen_yellow)
+
+# .............................................................................
+# letter G
+# .............................................................................
+# horizontal red line
+fig.plot(x=[0.1, 1.65], y=[0, 0], pen=f"12p,{color_red}")
+# red ring sector
+fig.plot(x=0, y=0, style="w3.3c/90/0+i2.35c", fill=color_red)
+# space between yellow lines and ring sector
+fig.plot(x=0, y=0, style="w3.7c/0/360+i3.3c", fill=color_bg)
+# vertical yellow line
+fig.plot(x=[0, 0], y=[-4, 4], pen=f"6p,{color_yellow}")
+# cover yellow line in lower part of the ring sector
+fig.plot(x=0, y=0, style="w3.3c/260/-80+i2.35c", fill=color_red)
+
+# .............................................................................
+# upper vertical red line
+# .............................................................................
+# space between red line and blue circle / hexagon
+fig.plot(x=[0, 0], y=[4.01, 3.0], pen=f"18p,{color_bg}")
+# red line
+fig.plot(x=[0, 0], y=[4.00, 1.9], pen=f"12p,{color_red}")
+
+# .............................................................................
+# letter M
+# .............................................................................
+# space between letter M and yellow line at the right side
+fig.plot(x=[1.6, 1.6], y=[1.5, 1.775], pen=f"10p,{color_bg}")
+# fig.plot(x=[1.6, 1.6], y=[1.5, 2.0], pen=f"10p,{color_bg}")
```

Comment:
Another detail is the ending of the yellow line at the right sight of the letter M:

| using line 146 | using line 147 |
| --- | --- |
| ![pygmt_logo_m_corner_02](https://github.com/user-attachments/assets/713a92fe-c12d-489c-8b99-4fca22f49921) | ![pygmt_logo_m_corner_01](https://github.com/user-attachments/assets/25ed216a-c3c2-4447-b71c-b51befb23f5c) |

Reply:
Currently, the code is now using the right version (see commit https://github.com/GenericMappingTools/pygmt/pull/3849/commits/9bbf4433de59d18f9d8435a55b993619d9eecc99).